src/main/kotlin/blackjack/Game.kt

```diff
@@ -0,0 +1,41 @@
+package blackjack
+
+import blackjack.domain.Deck
+import blackjack.domain.Player
+import blackjack.io.Input
+import blackjack.io.Output
+
+class Game(val input: Input, val output: Output) {
+    lateinit var players: List<Player>
+    private var deck: Deck = Deck()
+
+    fun init() {
+        players = input.getPlayers()
+            .map { Player(it) }
+
+        deck.shuffle()
+    }
```

Comment:
```suggestion
    private val players: List<Player> by lazy {
        makePlayers()
    }

    private fun makePlayers() = input.getPlayers().map { Player(it) }
```

player에 대해서 지연 초기화(lazy initialization) 패턴을 사용해볼까요?

링크
- [Lazy properties﻿](https://kotlinlang.org/docs/delegated-properties.html#lazy-properties)